### PR TITLE
Add e2e tests workflow

### PR DESCRIPTION
**Pull request by kravciak**
Local run: https://github.com/kravciak/ui/actions/runs/4323509003/jobs/7547196524

### CODE REVIEW

**Review by viccuad — APPROVED** [review]
From the GHA side of things it looks good to me!

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+name: Playwright Tests
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - '*'
+  schedule:
+    - cron: "0 2 * * *"
+
```
Comment (jordojordo) [inline .github/workflows/playwright.yml:9]: Should we add a dispatch for `pull_request` as well?

**Review by jordojordo — COMMENTED** [review]
One question - otherwise looks good to me

File: .github/workflows/playwright.yml (hunk 1 above)
Comment (kravciak) [reply .github/workflows/playwright.yml:9]: Now I install latest available tag - so it does not make sense on PRs if they are not tagged.
I would like to have some quick workflow for PRs and more complex for tags, but it's just an idea for now.

**Pull request merged**